CONTRIBUTING.md

```diff
@@ -92,7 +92,7 @@ We also ship the type declaration files for all TypeScript components to ensure
 Our current supported React version is 17.0.2, please be mindful when using React features that may not yet be supported.
 
 #### Design
-If you'd like to contribute a change to a React component, please first reach out to Koala team on Slack to discuss and agree on the proposed change. Make sure to add to your message the design (Figma file) and include examples for each state e.g. disabled, expanded etc. if applicable.
+If you'd like to contribute a change to a React component, please first reach out to the backpack team on Slack to discuss and agree on the proposed change. Make sure to add to your message the design (Figma file) and include examples for each state e.g. disabled, expanded etc. if applicable.
```

Comment:
Does contacting backpack via slack also apply to OSS contributors?

Reply:
No this is done via issues, but here we don't need to call this out because for design related, our designers aren't on OSS so this part is not applicable here

Reply:
Makes sense.